client_src/js/dashboard.runner.js

```diff
@@ -67,7 +68,68 @@ async function initNewGraphs() {
   await ChartsManager.renderPrintCompletionByDay(printCompletionByDay);
 }
 
-loadClientSSEWorker();
+const workerURL = "/dashboardInfo/get/";
+
+async function workerEventFunction(data) {
```

Comment:
I'd like to keep this stuff split from the dashboard entry file, see my review comment in the worker file. I could've named this file `dashboard.entry.js`. The intention is that it binds everything together but little code is present.